content/docs/Configuration/_index.md

```diff
@@ -21,13 +21,67 @@ Both Packit Service and Packit CLI use this configuration file.
 Here you can see a list of options that can be defined at the top-level and
 shared by all of the jobs.
 
+#### packages
+
+(*dict*) This key was introduced mainly to enable support for monorepositories, upstream repositories
+mapping to multiple downstream packages. It holds a dictionary of `{<package_name>: <package_object>}`.
+
+A configuration utilising the `packages` key may look like this:
+```yaml
+packages:
+
+  python:
+    downstream_package_name: python-copr
+    upstream_package_name: copr
+    paths: 
+      - ./python
+    specfile_path: python-copr.spec
+    files_to_sync: 
+      - python-copr.spec
+
+  cli:
+    downstream_package_name: copr-cli
+    upstream_package_name: copr-cli
+    paths: 
+      - ./cli
+    specfile_path: copr-cli.spec
+    files_to_sync: 
+      - copr-cli.spec
+```
+
+If the configuration doesn't include the `packages` key, it is expected that there is only one package
+and the package-specific options can be defined on the top level:
+
+```yaml
+downstream_package_name: python-copr
+upstream_package_name: copr
+paths: 
+  - ./python
+specfile_path: python-copr.spec
+files_to_sync: 
+  - python-copr.spec
+```
+
+#### jobs
+
+(*list of dicts*) A list of job definitions for Packit Service: see
+[Packit Service jobs configuration](jobs) for details.
+
+### Package-specific keys
```

Comment:
```suggestion
### Package-specific keys

All packit configuration keys can be used as `package` keys (exception made for `upstream_project_url` which has to appear just once).
```

Reply:
I would move this sentence and the `paths` related docs just belove `packages` (before `jobs`), because they are not top-level keys. But are used under `package`.

Reply:
thanks for the suggestions! I rewrote it a little bit, PTAL